### PR TITLE
Update docs to reflect #222.

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -466,6 +466,8 @@ end
 ```
 
 ### windows_reboot
+This resource is now deprecated.  Use the built-in [reboot](http://docs.chef.io/resource_reboot.html) resource.
```
Comment (btm) [inline README.md]: When was reboot added?

**Comment by btm** [comment]
"Chef >= 11.6.0 includes a built-in" <- in this its clear the new resource comes in the core Chef client
"Use the built-in" <- in this one I'm not sure what it is built into.

**Comment by btm** [comment]
You might want to say "This resource is deprecated and will be removed in a future version of this cookbook." for those people who don't talk about deprecation daily.

File: README.md (hunk 1 above)
Comment (ksubrama) [inline README.md]: ¯\_(ツ)_/¯ 

I'll go through the changelog and find a version,

**Comment by ksubrama** [comment]
Done!

**Pull request merged**